source/MaterialXFormat/XmlIo.h

```diff
@@ -19,38 +19,61 @@ namespace MaterialX
 /// @name Reading
 /// @{
 
+/// @struct XmlReadOptions
+/// Structure to hold options to use during reading. 
+///
+struct XmlReadOptions
+{
+    XmlReadOptions() :
+        readXincludes(true),
+        skipDuplicates(false) {}
+
+    virtual ~XmlReadOptions() {}
+
+    /// If true, XInclude references will be read from disk and included in the document. Defaults to true.
+    bool readXincludes;
+    /// If true, Will skip reading in Elements with duplicate names. Defaults to false.
+    bool skipDuplicates;
+};
+
 /// Read a document as XML from the given character buffer.
 /// @param doc The document into which data is read.
 /// @param buffer The character buffer from which data is read.
+/// @param readOptions Options to use during reading. Default is null which indicates to use the 
+/// default values as specified in the XmlReadOptions structure.
```

Comment:
Formatting suggestion:  In the codebase so far, we've been starting the second and following lines of a "param" description with four spaces, to make the text more readable.